content/en/docs/release-oversight/disruption-testing/job-primer.md

```diff
@@ -9,7 +9,17 @@ description: Job Primer ci-tool used to generate job names for Big Query.
 In `Job Primer` a job name is very important. Please make sure that the job names contain correct information. ([see options below](#naming-convention))
 {{% /alert %}}
 
-[JobPrimer](https://github.com/openshift/ci-tools/tree/master/pkg/jobrunaggregator/jobtableprimer) is the `ci-tool` that is used to populate the `BigQuery` `Jobs` table. The `Jobs` table is what dictates the periodic jobs are grabbed during `disruption` data gathering. Currently this tool is ran manually.
+[JobPrimer](https://github.com/openshift/ci-tools/tree/master/pkg/jobrunaggregator/jobtableprimer) is the `ci-tool` that is used to populate the `BigQuery` `Jobs` table. The `Jobs` table is what dictates the periodic jobs are grabbed during `disruption` data gathering.
+
+`JobPrimer` is ran periodically in our CI. There are two sub command described below that run on a cron.
```

Comment:
Consider:
`JobPrimer` runs periodically in our CI.

Reply:
Sounds good, updated